sql/moz-fx-data-shared-prod/mozilla_org_derived/www_site_downloads_v1/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+friendly_name: WWW Site Downloads
+description: |-
+  Aggregated download counts for www.mozilla.org
+owners:
+- ascholtz@mozilla.com
```

Comment:
@kwindau since you're creating this model, are you also meant to be in the owners list? Does Anna know you're adding her here?

Reply:
Good call I just copied the old owner, but can update it/switch it to me